### DISCUSSION TITLE
POLL - Nav Launch with Throttle Low option

### DISCUSSION THREAD

**Discussion by breadoven** [iNavFlight/inav#7973] [Polls]
Poll to get feedback on interest in introducing the option outlined in https://github.com/iNavFlight/inav/pull/6389, Nav Launch with throttle low.

**Comment by rts18** [comment]
Not bothered either way. 
If it were not configurable I would be against it.

**Comment by MrD-RC** [comment]
I’m not keen on the motor starting as soon as armed if you have motor stop or idle throttle enabled.

**Comment by carlybaby69** [comment]
We arm all the time, in our houses, in the field. I like the extra safety net of raising the throttle when you’re actually ready for the real thing. Plus I like to have the throttle set to where I think I’m going to need it after my autolaunch goes wrong and I have / want to manually take over. What I’m saying is I like my throttle high on an autolaunch! Like a TOGA!
